src/gda/backend_gda.hpp

```diff
@@ -96,6 +91,13 @@ class GDAHostContext;
 class QueuePair;
 class HostInterface;
 
+enum GDA_NIC {
+  INVALID,
```

Comment:
Is `INVALID` the best name for this? It seems to be used as a sort of 'auto-select' mechanism that tries each one progressively. If there's a need for a type tag representing that there is no valid GDA device, I'd recommend `NONE`.
```suggestion
  NONE,
  AUTO,
```